lib/screens/home/accounts/multi_account_item.dart

```diff
@@ -162,13 +162,6 @@ class _MultiAccountItemState extends State<MultiAccountItem> {
                   Navigator.of(context).pop(MenuOperation.select);
                 },
               ),
-            PwListDivider(),
-            PwGreyButton(
-              enabled: false,
-              text: '',
-              // ignore: no-empty-block
-              onPressed: () {},
-            ),
```

Comment:
This wasn't doing anything, true, but it was there to match the design. Does this change the way the menu looks? If so, this needs to be reverted. Or at least changed so it matches the way it used to look.

Reply:
on smaller screens it caused the menu to be partially scrolled. This obscured part of the available actions to show blank space